backend/routes/loginAuth.ts

```diff
@@ -1,26 +1,21 @@
-import { type User } from "@prisma/client";
-import express from "express";
+import { type User } from '@prisma/client';
+import express from 'express';
 import passport from 'passport'
 import prisma from '../utils/prismaClient'
 
 const loginAuthRouter = express.Router()
-const db = prisma;
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
should we disable this line or should we actually think about why we want to keep an unused variable?